### ISSUE TITLE
Casi particolari di segnalazioni e linee guida per approvazione (su Facebook)

### ISSUE THREAD

**Issue by Albertopi78** [emergenzeHack/covid19italia#142]
Su Facebook è arrivata questa segnalazione.

<img width="500" alt="Schermata 2020-03-13 alle 13 22 01" src="https://user-images.githubusercontent.com/42850988/76620534-ac7c9180-652d-11ea-9aba-259d60206d57.png">

Si tratta di fatto di informazioni utili che però sono già spiegate sui siti ufficiali.
Il mio dubbio nell'approvare segnalazioni simili sta nel fatto che non possiamo metterci a verificare la correttezza delle risposte alle domande. Che ne dite? 
Personalmente, eviterei.

**Comment by luciaroma** [comment]
io mi sa che concordo con te

**Comment by Albertopi78** [comment]
Preparo una risposta standard per questo tipo di segnalazioni, se vi va bene.  
«Ciao! Grazie di cuore per la tua segnalazione. Per quanto riguarda domande e risposte che riguardano decreti, ordinanze, leggi e interpretazioni dei medesimi, però, preferiamo rimandare alla lettura delle fonti ufficiali. Grazie ancora!»

**Comment by avivace** [comment]
Concordo, non ha senso e non aggiunge valore se non di story telling, incorrendo in rischi inutili
